Jetcaster/wear/src/main/res/drawable/ic_launcher_foreground.xml

```diff
@@ -0,0 +1,31 @@
+<!--
+  ~ Copyright 2020 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Should we remove the assets from the `app` module to `designsystems` (or a new module) so that these can be shared? Or does Wear use a separate launched icon? Related PR: https://github.com/android/compose-samples/pull/1291

Reply:
I think we can use the same, we can take a look once the other PR is merged